src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.CSharpCodeGenerator.cs

```diff
@@ -609,12 +611,23 @@ protected override async Task<GeneratedCode> CreateGeneratedCodeAsync(OperationS
                     // here, we explicitly insert newline at the end of "{" of auto generated method decl so that anchor knows how to find out
                     // indentation of inserted statements (from users code) with user code style preserved
                     var root = newDocument.Root;
-                    var methodDefinition = root.GetAnnotatedNodes<MethodDeclarationSyntax>(this.MethodDefinitionAnnotation).First();
-
-                    var newMethodDefinition = TweakNewLinesInMethod(methodDefinition);
```

Comment:
feels like you can keep the old code, but just update TweakNewLinesInMethod

Reply:
I think that even if we added to the same method, then the length of the method would pretty much remain the same since I don't think much of the code can be re-used (due to MethodDeclarationSyntax and LocalFunctionStatementSyntax having dissimilar base classes).